components/colorloupe/metadata/colorloupe.yml

```diff
@@ -1,19 +1,34 @@
 name: Color Loupe
 status: Verified
-SpectrumSiteSlug: https://spectrum.adobe.com/page/color-area/
+SpectrumSiteSlug: https://spectrum.adobe.com/page/color-loupe/
 description: |
   - Set the `--spectrum-picked-color` custom property to the CSS color value you want to show
   - Apply `.is-open` to display the loupe
   - Color Loupe does not have a disabled style; do not show it when the handle its attached to is disabled.
 examples:
-  - id: color-area
+  - id: color-loupe
     name: Standard
     demoClassName: spectrum-CSSExample-example--spacious
     markup: |
-      <svg class="spectrum-ColorLoupe is-open" style="--spectrum-picked-color: rgba(255, 0, 0, 0.5); position: absolute; top: 25%; left: 50%;">
+      <svg class="spectrum-ColorLoupe is-open" overflow="visible" style="--spectrum-picked-color: rgba(255, 0, 0, 0.5); position: absolute; inset-block-start: 25%; inset-inline-start: 50%;">
```

Comment:
`overflow="visible"` <- legit did not know that worked on SVGs! #TIL

Reply:
These components have been a lot of SVG learnings 😆